### PR TITLE
[Group] Update group demo to include encryption configuration

### PR DESCRIPTION
**Pull request by mkardous-silabs**
#### Problem
* Group key management cluster was not enabled on lighting app.
* Group encryption was merged and configuration wasn't done in the group demo config yaml

#### Change overview
* Enable group key management cluster on lighting app
* Add keys configuration to group demo config

#### Testing
* Manual test on efr32 to validate the demo works

### CODE REVIEW

**Review by woody-apple — APPROVED** [review]

File: examples/lighting-app/lighting-common/lighting-app.zap [hunk 1]
```diff
@@ -4509,6 +4509,152 @@
...
+            }
+          ],
+          "attributes": [
+            {
+              "name": "groupKeyMap",
+              "code": 0,
+              "mfgCode": null,
+              "side": "server",
+              "included": 1,
+              "storageOption": "RAM",
```
Comment (tcarmelveilleux) [inline examples/lighting-app/lighting-common/lighting-app.zap:4587]: Shouldn't all the `storageOption` be set to "External" ?

File: src/app/tests/suites/TestGroupDemoConfig.yaml [hunk 2]
```diff
@@ -40,12 +40,37 @@ tests:
...
+                        GroupKeySecurityPolicy: 0,
+                        EpochKey0: "\xa0\xa1\xa2\xa3\xa4\xa5\xa6\xa7\xa8\xa9\xaa\xab\xac\xad\xae\xaf",
+                        EpochStartTime0: 1110000,
+                        EpochKey1: "\xb0\xb1\xb2\xb3\xb4\xb5\xb6\xb7\xb8\xb9\xba\xbb\xbc\xbd\xbe\xbf",
+                        EpochStartTime1: 1110001,
+                        EpochKey2: "\xc0\xc1\xc2\xc3\xc4\xc5\xc6\xc7\xc8\xc9\xca\xcb\xcc\xcd\xce\xcf",
+                        EpochStartTime2: 1110002,
+                    }
+
+    - label: "Write Group Keys"
```
Comment (tcarmelveilleux) [inline src/app/tests/suites/TestGroupDemoConfig.yaml:71]:
```suggestion
    - label: "Map Group Key Set to group ID on a given fabric"
```

**Review by tcarmelveilleux — APPROVED** [review]

**Review by woody-apple — APPROVED** [review]

**Review by bzbarsky-apple — APPROVED** [review]

File: examples/lighting-app/lighting-common/lighting-app.zap (hunk 1 above)
Comment (bzbarsky-apple) [reply examples/lighting-app/lighting-common/lighting-app.zap:4587]: Fwiw, for a list we just ignore what's in the .zap file and use "External" anyway.  But the non-list attributes should have "External" if they don't need RAM storage.

**Review by jmartinez-silabs — APPROVED** [review]

**Pull request merged**